nagios.yml

```diff
@@ -136,3 +138,7 @@ parameters:
   name: "NAGIOS_PASSWORD"
   generate: "expression"
   from: "[a-zA-Z0-9]{10}"
+-
+  description: "Mbaas router dns"
```

Comment:
A better description might be something like 'Host or Service name for the MBaaS i.e. the host that fh-mbaas is exposed as or the name of the fh-mbaas service'